src/print.c

```diff
@@ -876,6 +876,8 @@ static void print_super_block(struct scoutfs_super_block *super, u64 blkno)
 	print_block_header(&super->hdr, SCOUTFS_BLOCK_SM_SIZE);
 	printf("  format_hash %llx uuid %s\n",
 	       le64_to_cpu(super->format_hash), uuid_str);
+	printf("  blkdev type: %s\n",
```

Comment:
The printers tend to just be a struct dump, not so much translating the contents.  So let's just add a hex print of the flags somewhere by the other fields in the struct that it's near?   (Overall the print output could really use a stylistic refresh, but, ugh.)